packages/grafana-toolkit/src/cli/utils/execLine.ts

```diff
@@ -0,0 +1,28 @@
+import { exec } from 'child_process';
+import { getPluginJson } from '../../config/utils/pluginValidation';
+import { getCiFolder } from '../../plugins/env';
+import { getPluginId } from '../../config/utils/getPluginId';
+import path = require('path');
+
+const ciDir = getCiFolder();
+const distDir = path.resolve(ciDir, 'dist');
+const distContentDir = path.resolve(distDir, getPluginId());
+const pluginJsonFile = path.resolve(distContentDir, 'plugin.json');
+const pluginJson = getPluginJson(pluginJsonFile);
```

Comment:
weird that this is happening in the root context -- does it really make sense here?

Reply:
NOTE: this is also why the circle tests are failing